### ISSUE TITLE
Inserting duplicates of a frame range

### ISSUE THREAD

**Issue by KashouC** [aseprite/aseprite#1694]
I'm still kind of confused as to the way duplicate frame works. In some instances it's really helpful that it overwrites the cel on the next frame, but at the same time sometimes I want to duplicate multiple frames and have them be inserted right after, but there doesn't seem to be a good way to do that other than spamming new frame a couple of times and then duplicating to overwrite the new ones.

So what I was thinking is that while Duplicate Cel(s) works just fine as it is now, as New Frame creates a copy of the current frame in the position after it, New Frame could be New Frame(s), and create a copy of the currently selected frame range instead of just one frame.

**Comment by dacap** [comment]
Should be the frames duplicated right after the last selected frame?

**Comment by KashouC** [comment]
Yeah just when you have one frame selected, you get the copy of it inserted after your current "selection" (technically is more like the focused frame I guess, but still), so if you have 4 frames selected when you press new frame, you'd get a copy of those 4 frames inserted after the last selected frame. Might wanna change it into New Frame(s) if you implement it this way I guess.
